governance/default.go

```diff
@@ -622,14 +622,11 @@ func (g *Governance) initializeCache() error {
 	g.idxCache = indices
 
 	// Put governance items into the itemCache
-	for _, v := range indices {
-		if data, err := g.db.ReadGovernance(v); err == nil {
-			data = adjustDecodedSet(data)
-			g.itemCache.Add(getGovernanceCacheKey(v), data)
-			g.actualGovernanceBlock.Store(v)
-		} else {
-			logger.Crit("Couldn't read governance cache from database. Check database consistency", "index", v, "err", err)
-		}
+	if v, result, err := g.ReadGovernance(indices[len(indices)-1]); err == nil {
```

Comment:
@yoomee1313 Are all governance items are stored at the latest block?

Reply:
@jeongkyun-oh i think all items are stored at the latest block, but i restored the `indice iteration` just in case. Test result is same as non-iteration. It is reflected at https://github.com/klaytn/klaytn/pull/1117/commits/f70e2003dffeb4c742e02e1a49861f03d6e61cfb.